server/controllers/user-controller.js

```diff
@@ -0,0 +1,82 @@
+import {
+  successResponse,
+  errorResponse,
+  validate,
+  validationErrorResponse
+} from '../utils/helpers';
+import db from '../models';
+import { profileSchema } from '../utils/validators';
+
+const { User } = db;
+
+/**
+ * The controllers for users route
+ *
+ * @class UsersController
+ */
+class UsersController {
+  /**
+   * Create user profile
+   * @static
+   * @param {Request} req request object
+   * @param {Response} res response object
+   * @memberof {Users}
+   * @returns {undefined}
+   */
+  static async updateUser(req, res) {
+    validate(req.body, profileSchema)
+      .then(async () => {
+        const { id } = req.user;
+        try {
+          const { email, username, password, bio, image } = req.body;
+          const user = await User.findOne({
+            where: { id }
+          });
+          const data = await user.update(
+            {
+              email: email || user.email,
+              username: username || user.username,
+              password: password || user.password,
```

Comment:
Please don't change the user password here - there's a story for that.

Reply:
I will create another route for update